modules/ROOT/pages/batch-filters-and-batch-aggregator.adoc

```diff
@@ -1,17 +1,12 @@
-= Refining Batch Steps Processing
+= Using Batch Processing Components
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :page-aliases: batch-aggregator-concept.adoc, filter-records-batch-faq.adoc
 
-You can refine the work that a batch step performs upon the records it processes.
```

Comment:
This section talked about only 2 of several topics on this page. So it needed to be made more comprehensive. In the HTML, the main topics will be H2s that will have links on the right side of the page, so it isn't necessary to detail each one here.